test/mocks/mockLibThymeleaf.ts

```diff
@@ -0,0 +1,23 @@
+import {jest} from '@jest/globals';
+// import sp from 'synchronized-promise'
+import {TemplateEngine} from 'thymeleaf';
```

Comment:
No we cannot execute a Java library in node...

Reply:
I wouldn't be worried about dev dependencies, but that's me.

But if you look at getMetaData.test.ts, that thymeleaf render ins't really working, since it just returns the template as is without replacing the variables. So either I should get it working, or just be happy with the template being returned.